smoke-tests/apps/TelemetryProcessors/src/smokeTest/java/com/microsoft/applicationinsights/smoketest/TelemetryProcessorsTest.java

```diff
@@ -54,13 +53,23 @@ void doSimpleTestPiiData() throws Exception {
     assertThat(telemetry.rd.getProperties()).containsEntry("attribute2", "testValue2");
     assertThat(telemetry.rd.getProperties()).containsEntry("sensitiveAttribute1", "redacted");
     assertThat(telemetry.rd.getProperties().get("httpPath"))
-        .isEqualTo("*/TelemetryProcessors/sensitivedata*");
+        .isEqualTo("/TelemetryProcessors/sensitivedata");
     assertThat(telemetry.rd.getProperties()).hasSize(5);
     assertThat(telemetry.rd.getProperties())
         .containsEntry("_MS.ProcessedByMetricExtractors", "True");
     assertThat(telemetry.rd.getSuccess()).isTrue();
   }
 
+  @Test
+  @TargetUri("/user/123")
+  void shouldMask() throws Exception {
+
+    Telemetry telemetry = testing.getTelemetry(0);
+
+    assertThat(telemetry.rd.getProperties().get("httpPath"))
+        .isEqualTo("/TelemetryProcessors/user/**");
```

Comment:
I think `telemetry.rd.getUrl()` is probably the more important thing to test/verify (which I think will require masking the otel attribute `http.path`)

Reply:
Masking of the` http.url `attribute is tested [here](https://github.com/microsoft/ApplicationInsights-Java/blob/e701d955c6c9dc2ef85dba5c501121e9d3e5a80b/agent/agent-tooling/src/test/java/com/microsoft/applicationinsights/agent/internal/processors/TelemetryProcessorMaskingTest.java#L35) without the help of a Docker container.

I have also added the verification to the smoke test.